plugins/dashboard/frontend/src/app/components/ValuePayload.tsx

```diff
@@ -36,12 +36,12 @@ export class ValuePayload extends React.Component<Props, any> {
                 <Row className={"mb-3"}>
                     <Col>
                         <ListGroup>
-                            <ListGroup.Item>Parent 0: {payload.parent_id_0} </ListGroup.Item>
+                            <ListGroup.Item>Parent 0: {payload.parent1_id} </ListGroup.Item>

```

Comment:
If we call them `parent1` and `parent2` then we should probably also call them that way in the frontend?
```suggestion
                            <ListGroup.Item>Parent 1: {payload.parent1_id} </ListGroup.Item>

```